src/CHARTER.md

```diff
@@ -0,0 +1 @@
+# Charter
```

Comment:
It'd be good to open an issue and maybe link it here so we remember to write down the group's charter.

Reply:
Good point, done https://github.com/rust-lang/wg-macros/issues/2